ts-client/src/dlmm/index.ts

```diff
@@ -3263,7 +3264,8 @@ export class DLMM {
     swapForY: boolean,
     allowedSlippage: BN,
     binArrays: BinArrayAccount[],
-    isPartialFill?: boolean
+    isPartialFill?: boolean,
+    returnExtraBinArrays?: boolean
```

Comment:
may be change params to extraBinArrays with default value is 0, then user can specify how many binArrays they want to include

Reply:
updated the param to maxExtraBinArrays